helix-view-aggregator/src/main/java/org/apache/helix/view/aggregator/HelixViewAggregator.java

```diff
@@ -191,14 +191,19 @@ private void handleSourceClusterEvent(ClusterViewEvent event) {
         _refreshViewCluster.set(true);
         break;
       case PeriodicViewRefresh:
-        if (!_refreshViewCluster.get()) {
+        // refresh local view cluster data cache
+        boolean cacheChanged = _viewClusterRefresher.refreshViewClusterDataCache();
```

Comment:
just curious, refreshViewCluster can be set using the above call directly, correct?

Reply:
yes, other types of incoming event will set it.